services/sidecar/src/sidecar/_deprecated.py

```diff
@@ -95,8 +95,11 @@ def _process_task_input(self, port, input_ports):
                 other_task = None
                 _session = self._db.Session()
                 try:
-                    other_task =_session.query(ComputationalTask).filter(and_(ComputationalTask.node_id==other_node_id,
-                                    ComputationalTask.project_id==self._task.project_id)).one()
+                    # pylint: disable=no-member
+                    other_task =_session.query(ComputationalTask).filter(
```

Comment:
so this guy also uses the old ways... are you keeping the old one finally?

Reply:
I talked to @mguidon and we will keep it like that for the moment. Will be slowly removed